<launch>
    <include file="$(find feature-detect)/launch/usb_cam.launch"/>

    <node name="feature_detect" pkg="feature-detect" type="orb_features.py" output="screen"/>

    <!-- <node name="camera_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/usb_cam/image_raw"/>
        <param name="autosize" value="true" />
    </node> -->

    <node name="feature_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/feature_detector/orb_detection_image"/>
        <param name="autosize" value="true" />
    </node>
</launch>